src/main.rs

```diff
@@ -9,11 +9,12 @@ mod arg_parse;
 
 #[derive(Deserialize, Debug)]
 struct Config {
-    package: Package
+    package: Option<MetadataSection>,
+    workspace: Option<MetadataSection>
```

Comment:
Could we make this a serde enum, to encode at compile time that this must be either a package or a workspace (rather than allowing them to potentially both be `None`)?

Reply:
Of course, I hadn't thought about it. I'll make the changes!